0007/README.md

```diff
@@ -0,0 +1,221 @@
+    **expected** to ensure the health of the project overall, drive the overall
+    project strategy, and serve as an escalation point in the event of a
+    disagreement.
+    *   These duties are much less frequent than those of a Maintainer.
+    *   We **expect an SC Member to be well versed in the project**. This often
+        means that SC Members are also Core Maintainers, or maintainers of a
+        subproject, but this is not required.
+    *   Some Core Maintainers might not be SC Members.
+    *   The SC, as a group, are responsible for maintaining the relationship
+        with the SCI WG, which includes:
+        *   Facilitating input from SCI WG into SLSA during significant 
+            developments, including: adding tracks and establishing roadmaps
+        *   Periodically reporting progress and developments to SCI WG 
+    *   As community leaders, SC Members are also responsible for maintaining
+        the slsa-framework/governance repository.
+
+### Clarify GitHub roles and how they maps to project roles
+
+Currently it’s unclear how the GitHub org and repo membership and roles
+correspond to the roles defined in the governance document. The proposal here
+attempts to clarify.
+
+*   GitHub Organization:
+    *   Member:
+        *   Used for Maintainers to allow them to be added to a team.
+        *   Used as needed for Participants and Contributors who would like to
+            set up [custom notification routing] and issue assignment across
+            multiple repos.
+        *   Does not grant any special repository permissions, i.e. Base
+            permissions = “No permission”
+    *   Owner:
+        *   Used for SC Members, OpenSSF administrators, and (as needed)
+            Maintainers
+    *   Teams: (all with public visibility)
+        *   [SLSA Steering Committee][SLSA Steering Committee Team]
+        *   Core Maintainers
+        *   \<subproject\> Maintainers
+        *   Other teams as needed
+    *   GitHub Repositories
+        *   Read: may be granted to Participants and Contributors as needed,
+            usually to allow the person to set up custom notification routing
+            and/or be assigned issues
+        *   Triage/Write: may be granted to Contributors who have been
+            delegated this responsibility by a Maintainer
+            *   Write permission grants ability to approve PRs
+        *   Admin: granted to every Maintainer via the \<subproject\>
+            Maintainers team
+    *   These permissions should be reviewed and cleaned up regularly (e.g.
+        annually)
+
+Once this proposal is accepted, the following changes will be made to the
+GitHub organisation:
+
+*   Set up Maintainers teams and add Maintainers as members if needed
+*   Change the org member base permissions to “No permissions” (from “Admin”)
+    and check that everyone still has necessary access
+*   Purge unnecessary permissions in the org and in each repo
```

Comment:
may want to add a cadence.. quarterly.. every 6 months..etc.

Reply:
Good call, I'll add a note that we should do this regularly and at a minimum following each SC nomination cycle.

Reply:
Added in https://github.com/slsa-framework/slsa-proposals/pull/15/commits/4d91b1ab5415d562e4c3560d3edbbba923070d79